elastic/kibana
# Rename customMetric to metrics in the custom threshold rule criteria

Closes #163819

## Summary

This PR renames customMetric to metrics in the custom threshold rule criteria to have a cleaner API for this rule, considering that custom aggregation is now the default option.

<img src="https://github.com/elastic/kibana/assets/12370520/8953e7ad-aa2b-4bf6-b568-ff269ab84ca5" width=300/>


### 🧪 How to test
- Create a custom threshold rule and ensure the preview is working fine
- Check the related saved object and ensure the data is saved as shown above
- Make sure the related alerts are generated as expected
